Libraries/LibWeb/CSS/StyleValues/StrokeDasharrayStyleValue.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/CSS/CSSStyleValue.h>
+
+namespace Web::CSS {
+
+class StrokeDasharrayStyleValue : public StyleValueWithDefaultOperators<StrokeDasharrayStyleValue> {
```

Comment:
Can we use `StyleValueList` instead of creating a new style value type for this property? I'm trying to keep the (large) number of style value classes under control. :sweat_smile:

The `<dasharray>` type is mentioned in the svg animations spec, but at a glance it looks like we don't need to be able to detect it, just a limited set of other types: https://svgwg.org/specs/animations/#complexDistances

Reply:
It's mentioned as a type in specs. Isn't it nice to mirror that in code?

Reply:
Moved to StyleValueList.

(For the tangent, I'm not aware of any properties using this type, so I'm punting on that for now.)